### PR TITLE
Exercise MobileRes & Modal Confirm

### PR DESCRIPTION
**Pull request by Hitenp1**
Exercises now work on mobile and can remove with modal confirmation

### CODE REVIEW

File: src/stylesheets/exerciselist.scss [hunk 1]
```diff
@@ -69,3 +69,46 @@
...
+}
+
+td.img_col {
+  padding-bottom: 1em;
+  padding-top: 1em;
+  width: 15em;
+}
+
+
+@media only screen and (max-width: 500px) {
```
Comment (mkewls) [inline src/stylesheets/exerciselist.scss]: Nit, but you should import 'screens' and use $lg

File: app/components/exercises/ExerciseList.js [hunk 2]
```diff
@@ -34,7 +66,7 @@ const ExerciseList = ({ user, exercises, deleteExercise }) => (
           else return 0;
         }).map( exercise =>
           <TableRow key={ exercise.id } selectable={false} className="exercise-row">
-            <TableRowColumn style={{"paddingBottom":"1em", "paddingTop": "1em", width: "15em"}}>
+            <TableRowColumn className="img_col">
               <img src={exercise.img_url}></img>
```
Comment (mkewls) [inline app/components/exercises/ExerciseList.js]: try using 'img-responsive' class. this currently is thumbnailed on mobile. should ideally take up screen width.

File: app/components/exercises/ExerciseList.js [hunk 3]
```diff
@@ -46,7 +78,7 @@ const ExerciseList = ({ user, exercises, deleteExercise }) => (
             <TableRowColumn style={{width:"10px", "paddingRight": "8em"}}>
```
Comment (mkewls) [inline app/components/exercises/ExerciseList.js]: can you place this on the upper right corner of the table? that would be consistent with our other uses of a delete button.

File: app/components/exercises/ExerciseList.js [hunk 4]
```diff
@@ -13,18 +13,50 @@ import { Table, TableHeader, TableHeaderColumn,
...
+      confirmOpen: false,
+      exerciseId: "",
+      userId: ""
+    }
+    this.dialogClose = this.dialogClose.bind(this);
+    this.dialogOpen = this.dialogOpen.bind(this);
+  }
+
+  dialogClose(){
+    this.setState({confirmOpen: false});
```
Comment (jakepeyser) [inline app/components/exercises/ExerciseList.js]: This and below function should contain a single `setState` call, this is needlessly overloading the state reonciler

File: src/stylesheets/exerciselist.scss [hunk 5]
```diff
@@ -69,3 +69,46 @@
...
+
+  .mobile_table tr.exercise-row {
+    td {
+      height: 100% !important;
+    }
+
+  }
+
+}
+
```
Comment (jakepeyser) [inline src/stylesheets/exerciselist.scss]: spaces, spaces, spaces...

**Comment by jakepeyser** [comment]
Different images/descriptions cause misalignment in all views:
image
image

**Comment by jakepeyser** [comment]
Also, looks like the styling changes broke the navbar on mobile:
image

**Pull request merged**